components/addPermissionUsingWebIdButton/index.jsx

```diff
@@ -99,12 +99,23 @@ export default function AddPermissionUsingWebIdButton({
 }) {
   const [anchorEl, setAnchorEl] = useState(null);
   const [access, setAccess] = useState(createAccessMap(true));
+  const [permissions, setPermissions] = useState(null);
   const classes = useStyles();
   const [disabled, setDisabled] = useState(false);
   const { accessControl } = useContext(AccessControlContext);
   const [agentId, setAgentId] = useState("");
   const [dirtyForm, setDirtyForm] = useState(false);
 
+  useEffect(() => {
+    if (!accessControl) {
+      setPermissions(null);
```

Comment:
With `permissions` defaulting to `null` anyways, is this line necessary?

Reply:
Looks like it is not 👍

Reply:
I think it's good to have in cases where permissions gonfrom null->something and then something->null again

Reply:
In this case though, it's when accessControl is not available, so it won't change to non-null, right? I might be missing something.

Reply:
It might be going from something to null if you open a resource that you have Control access to first, and then to one that you don't have Control access.

But I'm not 100% on the flow there, so I might be wrong.

Reply:
When logging out the permissions, they don't seem to change when I add/remove this line going from a resource with control access to one without and viceversa.